tools/ola_trigger/config.ypp

```diff
@@ -65,6 +67,12 @@ assignment :
         delete $1;
         delete $5;
       }
+    | WORD_LITERAL optional_whitespace '=' optional_whitespace INTEGER_LITERAL {
```

Comment:
Thanks. That's good to know. I take it you actually mean "INTEGER_LITERAL { $$ = ola::strings::IntToString($1); }" though.

Done but not tested yet.